draft-ietf-tls-esni.md

```diff
@@ -842,6 +857,28 @@ example, if a client-facing server services many backend origin hosts, only one
 of which supports some cipher suite, it may be possible to identify that host
 based on the contents of unencrypted handshake messages.
 
+Beyond these primary security and privacy goals, ECH also aims to hide, to some
+extent, (a) whether or not a specific server supports ECH and (b) whether or
+not a specific connection uses or negotiates ECH. In particular:
+
+1. ECH support is secret from passive attackers who do not know the ECH
```

Comment:
I would replace this enum list with a one-liner at the end of the preceding paragraph. E.g.,:

> This protocol aims to achieve both properties, assuming the attacker is passive and does not know the set of ECH configurations offered by the server. It does not achieve these goals for active attackers.

Reply:
Nice! I'll do that.